core/chains/evm/client/erroring_node_test.go

```diff
@@ -85,7 +85,8 @@ func TestErroringNode(t *testing.T) {
 	require.Equal(t, n.errMsg, err.Error())
 
 	require.Equal(t, "<erroring node>", n.String())
-	require.Equal(t, NodeStateUnreachable, n.State())
+	state, _ := n.State()
```

Comment:
Could it be cleaner to leave `n.State()` returning only the state, and having a new function `n.StateAndHeight()` that returns both? Since it seems that most callsites only need the state, not the height.

Reply:
I am fine to do that, as long as @jmank88 agrees. Just double checking to avoid extra round trips here.

Reply:
Fixed